app/views/admin/creator_settings/_form.html.erb

```diff
@@ -8,20 +8,24 @@
 </div>
 
 <div class="crayons-field mt-6 align-left">
-  <%= label_tag :logo_svg, class: "crayons-field__label" do %>
+  <div class="crayons-field__label relative">
     Logo
-    <span class="crayons-field__required crayons-tooltip" data-tooltip="This will set the logo for your Forem" aria-describedby="logo-subtitle"></span>
+    <span class="crayons-field__required crayons-hover-tooltip" data-tooltip="This will set the logo for your Forem"></span>
```

Comment:
I haven't been able to trigger this so far - I think the span is really small so I can't manage to hover it with the mouse

Reply:
I fixed this. Here's a recording.

https://user-images.githubusercontent.com/833231/142026594-162d991a-58ed-41d5-926f-be2c41a434b1.mov